src/mpid/ch4/netmod/ucx/ucx_init.c

```diff
@@ -253,7 +253,8 @@ int MPIDI_UCX_init_local(int *tag_bits)
     if (MPIDI_UCX_TAG_BITS > MPIR_TAG_BITS_DEFAULT) {
         *tag_bits = MPIR_TAG_BITS_DEFAULT;
     } else {
-        *tag_bits = MPIDI_UCX_TAG_BITS;
+        /* NOTE: need 1 bit for MPIDI_UCX_AM_TAG */
```

Comment:
We should be able to drop this now that the AM branch was merged.

Reply:
The commit is dropped. Also the configure commits are now squashed.